FAQ.md

```diff
@@ -0,0 +1,71 @@
+
+## With multiple workspaces on a single cluster, that implies pods from multiple tenants are on the same host VM. Does this mean privileged pods are forbidden to avoid cross contamination with host ports and host paths?
+
+We aren't quite there yet. Security controls are especially important at the multi-tenant level and we'd love to hear your use cases in this area.
+
+## Do the service provider operators/controllers live outside of kcp?
+
+Controller patterns are something we are actively working on defining better.  In general, operators and controllers from a service providing team would run on their provided compute (or shared compute) and point back to kcp in order to have a view of the resources being created that they need to take action upon. This view would be via a [Virtual Workspace](https://github.com/kcp-dev/kcp/blob/main/docs/virtual-workspaces.md).
+
+## Can I get the logs of a pod deployed via kcp? Can I attach to a pod or exec to a pod?
+
+Yes. We are tracking [read-through of resources](https://github.com/kcp-dev/kcp/issues/25) and [debugging](https://github.com/kcp-dev/kcp/issues/521) as use cases we need to support. You can view a demo of the current work in our (April 26 Community Call Recording)[https://www.youtube.com/watch?v=joR39RR2Gwo].
+
+## Are workspaces hierarchical?
+
+[Worksapces](https://github.com/kcp-dev/kcp/blob/main/docs/workspaces.md) can contain other workspaces and workspaces are typed. Please see the [Worksapce documentation](https://github.com/kcp-dev/kcp/blob/main/docs/workspaces.md) for more details.
+
+## Are custom admission controllers considered? How would that work across clusters if api server and the actual service is located elsewhere?
+
+Yes. [Validating and mutating webhooks](https://github.com/kcp-dev/kcp/pull/818) via an external URL are supported.
+
+## kcp hides nodes - does it mean that Pod does not have a nodeName field set?
+
+They do, in the [WorkloadClusters](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#workload-cluster) where the pods live and run. The control plane doesn't have pods (at least not by default, today).
+
+## Let’s take something boring like FIPS compliance. Would a workspace be guaranteed to run accordingly to the regulatory standards? Ie a workspace admin defined some FIPS stuffs and kcp ensures that the resulting pods do run appropriate in the FIPS shard?
+
+In kcp and application should be able to describe the constraints it needs in its runtime environment. This may be techincal requirements like GPU or storage, it may be regulatory like data locality or FIPS, or it may be some other cool thing we haven't thought of yet. kcp expects the integration with [Location and Placement](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#location) APIs to handle finding the right placement that fulfills those requirements.
```

Comment:
```suggestion
In kcp an application should be able to describe the constraints it needs in its runtime environment. This may be techincal requirements like GPU or storage, it may be regulatory like data locality or FIPS, or it may be some other cool thing we haven't thought of yet. kcp expects the integration with [Location and Placement](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#location) APIs to handle finding the right placement that fulfills those requirements.
```